frontend/__tests__/components/marketplace/kubernetes-marketplace.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+
+import {CatalogTile} from '../../../node_modules/patternfly-react-extensions/dist/js/components/CatalogTile';
+import {FilterSidePanel} from '../../../node_modules/patternfly-react-extensions/dist/js/components/FilterSidePanel';
+
+import { MarketplaceItemModal } from '../../../public/components/marketplace/marketplace-item-modal';
+import { MarketplaceListPage } from '../../../public/components/marketplace/kubernetes-marketplace';
+import { marketplaceListPageProps } from '../../../__mocks__/marketplaceItemsMocks';
+
+describe(MarketplaceListPage.displayName, () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(<MarketplaceListPage {...marketplaceListPageProps} />);
+  });
+
+  it('renders tiles correctly', () => {
```

Comment:
Can this be split into separate `it()` cases that are more specific/descriptive instead of just "renders tiles correctly"?

Reply:
Can do. :)